doc/api/deprecations.md

```diff
@@ -914,6 +914,14 @@ Type: Runtime
 
 This was never a documented feature.
 
+<a id="DEP0101"></a>
```

Comment:
Shouldn't the `0101` here and below be `XXXX` until the PR lands?

Reply:
Yep. `DEP00XX` until it lands. The person landing should assign the code